api/v1/apmalertcondition_types.go

```diff
@@ -0,0 +1,113 @@
+package v1
+
+import (
+	"encoding/json"
+
+	"github.com/newrelic/newrelic-client-go/pkg/alerts"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ApmAlertConditionSpec defines the desired state of NrqlAlertCondition
+type ApmAlertConditionSpec struct {
+	Terms               []NRAlertConditionTerm      `json:"terms,omitempty"`
+	Type                string                      `json:"type,omitempty"` //TODO: add conditionType or pull from alerts package or make string
```

Comment:
Is this TODO still accurate? Seemed like the validation uses the client's value objects, but maybe that's not what was intended by this message.

I feel that if there is work to be done, we should create an Issue instead of using code TODOs.

Reply:
it's not needed anymore, validation was moved to the admission webhook so I'll remove it